buildlib/azure-pipelines-pr.yml

```diff
@@ -252,19 +252,63 @@ stages:
               set -eE
               . buildlib/tools/common.sh
               prepare_build
-              ${WORKSPACE}/configure $(CONFIGURE_OPTS)
+              ${WORKSPACE}/configure $(CONFIGURE_OPTS) --prefix=$(Build.Repository.LocalPath)/install
               gcc -v
               make -s -j `nproc`
+              make install
+            displayName: Build ucx artifact
+          - task: CopyFiles@2
+            inputs:
+              sourceFolder: '$(Build.Repository.LocalPath)'
+              contents: |
+                install/**
+              targetFolder: '$(Build.ArtifactStagingDirectory)'
+          - task: PublishBuildArtifacts@1
+            inputs:
+              pathToPublish: '$(Build.ArtifactStagingDirectory)'
+              artifactName: ucx_bin_$(Build.BuildId)
+      - job: Test_in_containers
+        dependsOn: Distros
+        displayName: UCX tests in containers
+        pool:
+          name: MLNX
+          demands:
+          - ucx_docker -equals yes
+        strategy:
+          matrix:
+            ubuntu20:
+              CONTAINER: ubuntu2004
```

Comment:
You build on Centos and launch on Ubuntu? is it correct?

Reply:
Yes that's the goal of the test